scripts/deployment/deploy.sh

```diff
@@ -47,7 +49,18 @@ case $opt in
 					aws s3 cp s3://cloudcv-secrets/evalai/${env}/docker_${env}.env ./docker/prod/docker_${env}.env
 					docker-compose -f docker-compose-${env}.yml rm -s -v -f
 					docker-compose -f docker-compose-${env}.yml pull
-					docker-compose -f docker-compose-${env}.yml up -d --force-recreate --remove-orphans statsd-exporter django nodejs nodejs_v2 celery node_exporter push_gateway
+					docker-compose -f docker-compose-${env}.yml up -d --force-recreate --remove-orphans statsd-exporter django nodejs nodejs_v2 celery node_exporter
+				ENDSSH2
+                ssh ubuntu@${MONITORING_INSTANCE} -o StrictHostKeyChecking=no AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID} COMMIT_ID=${COMMIT_ID} env=${env} 'bash -s' <<-'ENDSSH2'
```

Comment:
Fix indentation

Reply:
![image](https://user-images.githubusercontent.com/55936223/125960259-509b506a-4e5d-4a28-81f1-29e99c94bdaf.png)
@Ram81 this looks like this in VS-code, I think the view here seems distored.... does the view of VScode seems fine?

Reply:
Your vscode settings might not be correct. It should show correct indentation in the PR. Please make sure you are using spaces in vscode to make sure it is consistent